### ISSUE TITLE
TypeError("unsupported operand type(s) for +: 'int' and 'str'")

### ISSUE THREAD

**Issue by kaiserdan** [Alex-Lekov/AutoML_Alex#34]
Is it intended that this error happens ?

```
  File "/home/../python3.7/site-packages/automl_alex/data_prepare.py", line 1050, in fit_transform
    data = self._clean_outliers_enc.transform(data)
  File "/home/../python3.7/site-packages/automl_alex/data_prepare.py", line 693, in transform
    feature_name = weight_values + "_Is_Outliers_" + self.method
TypeError: unsupported operand type(s) for +: 'int' and 'str'
```

**Comment by Alex-Lekov** [comment]
fix in version v1.3.10
thanks for noticing!. few people use zscore. I will add to my tests

**Comment by Alex-Lekov** [comment]
fixed [https://github.com/Alex-Lekov/AutoML_Alex/releases/tag/1.3.10](https://github.com/Alex-Lekov/AutoML_Alex/releases/tag/1.3.10)
